### PR TITLE
Disabling autorun of Python target

### PR DESCRIPTION
**Pull request by anton-malakhov**
Towards https://github.com/modin-project/modin/issues/1880
Excludes Python from ALL. Thus omniscidb can be build without python part and the python part can be built specifying Python as the target

### CODE REVIEW

File: Embedded/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@ IF(PYTHON)
         COMMAND cd ${CMAKE_CURRENT_BINARY_DIR} && ${PYTHON} setup.py build_ext -g -f -I ${CMAKE_CURRENT_SOURCE_DIR}
         )
 
-    add_custom_target(Python ALL DEPENDS ${OUTPUT})
+    add_custom_target(Python DEPENDS ${OUTPUT})
```
Comment (Garra1980) [inline Embedded/CMakeLists.txt:27]: What is the purpose of this change?

**Comment by anton-malakhov** [comment]
new updates are in #73
